### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#46639]
#### VFS team name
MOVE

#### VFS product name
1095B Tax Form

#### VFS feature name
1095B Tax Form - accessibility version

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
The heading "Choose your file format and download your document" is [coded as an H3](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/download-1095b/components/App/index.js#L110), and each of the alerts ([successful download](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/download-1095b/components/App/index.js#L189) and [failed download](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/static-pages/download-1095b/components/App/index.js#L161)) also have H3 headings.

From the Sketch prototype, it looks like these headings are always the next heading that follows the H1 on the page. They should be H2s rather than H3s.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

---

### Platform Recommendation
Change the H3 headings to H2 headings.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by joshkimux** [comment]
Hi @mdewey ! Would love your help on this ticket as part of our[ larger a11y dashboard effort](https://docs.google.com/spreadsheets/d/1lgzMiZZ-0rBPDuB2gRupBAjsyF0WTe005VvEHYrMxww/edit?usp=sharing).
- Is this issue blocked / have any dependencies?
- If not, who would be the best person to fix/resolve this?

Platform recommends to fix `defect-2` issues in 1-2 sprints post launch, but more importantly, this is a huge usability boost for Veterans using screen readers 🙇
